shared/profile/prove-website-choice/container.js

```diff
@@ -10,7 +10,7 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
   // both with a preference for https
   onCancel: () => dispatch(ProfileGen.createCancelAddProof()),
   onOptionClick: choice =>
-    dispatch(ProfileGen.createAddProof({platform: choice === 'file' ? 'https' : 'dns'})),
+    dispatch(ProfileGen.createAddProof({platform: choice === 'file' ? 'web' : 'dns'})),
```

Comment:
you can remove the comment above now i think

Reply:
would be nice to have an option for the user to specify http or https, although that could be a separate ticket